test/test_mps.py

```diff
@@ -8679,6 +8688,9 @@ def get_samples():
                 self.assertEqual(cpu_out, mps_out, atol=atol, rtol=rtol)
 
             except Exception as e:
+                if any(s in str(e).lower() for s in ["int64", "macos 13"]):
+                    self.skipTest(f"{str(e)}")
+
```

Comment:
This feel wrong to me. If we want to skip tests, we should explicitly decorate them with xfail!

Reply:
How do you suggest we do that for specific tests in TestConsistency? The purpose is to skip the tests which are expected to fail based on the error message from the TORCH_CHECKs. Do you know any example from which I can get an idea?

Reply:
In order to unblock this PR, I removed that part and will address catching the exceptions in a separate PR later.
This PR should still work without that exception-catching code.